src/entities/project.ts

```
@@ -752,3 +767,96 @@
+  @Field(_type => [Project], { nullable: true })
+  async projects(): Promise<Project[]> {
+    const causeProjects = await CauseProject.find({
+      where: { causeId: this.id },
+      relations: ['project'],
+    });
+    return causeProjects.map(cp => cp.project);
+  }
+
+  // Virtual field to get projects count
+  @Field(_type => Number, { nullable: true })
+  async activeProjectsCount(): Promise<Number> {
```

Biome (1.9.4)

[error] Don't use 'Number' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'number' instead


(lint/complexity/noBannedTypes)